### PR TITLE
fix(simmer, PID): fix simmer performance and tune parameters

### PR DESCRIPTION
**Pull request by Gaiejj**
## Description

#. Clean Simmer code style
#. Fix Simmer performance
#. Tune the parameters of PID algorithms

The performance of those algorithms can be seen at: https://wandb.ai/pku_rl/omnisafe/reports/OmniSafe-Mujoco-Velocity-Benchmarks--Vmlldzo0MDgzMDc3

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://github.com/OmniSafeAI/omnisafe/blob/HEAD/CONTRIBUTING.md) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.
- [ ] I have reformatted the code using `make format`. (**required**)
- [x] I have checked the code using `make lint`. (**required**)
- [x] I have ensured `make test` pass. (**required**)

### CODE REVIEW

**Comment by zmsn-2077** [comment]
The revision of the existing algorithm makes a lot of sense, but can the performance graph of  algorithm be added to this PR?

File: omnisafe/algorithms/on_policy/simmer/ppo_simmer_pid.py [hunk 1]
```diff
@@ -14,10 +14,11 @@
 # ==============================================================================
 """Implementation of the Simmer version of the PPO algorithm."""
```
Comment (zmsn-2077) [inline omnisafe/algorithms/on_policy/simmer/ppo_simmer_pid.py:15]: pid

File: omnisafe/algorithms/on_policy/simmer/trpo_simmer_pid.py [hunk 2]
```diff
@@ -14,10 +14,11 @@
 # ==============================================================================
 """Implementation of the Simmer version of the TRPO algorithm."""
```
Comment (zmsn-2077) [inline omnisafe/algorithms/on_policy/simmer/trpo_simmer_pid.py:15]: pid

File: omnisafe/algorithms/on_policy/simmer/trpo_simmer_pid.py [hunk 3]
```diff
@@ -83,10 +76,5 @@ def _update(self) -> None:
         """
         # note that logger already uses MPI statistics across all processes..
```
Comment (zmsn-2077) [inline omnisafe/algorithms/on_policy/simmer/trpo_simmer_pid.py:77]: del .

File: omnisafe/common/simmer_agent.py [hunk 4]
```diff
@@ -97,24 +104,31 @@ def get_greedy_action(
         # compute the error
         current_error = safety_budget - observation
         # blur the error
```
Comment (zmsn-2077) [inline omnisafe/common/simmer_agent.py:106]: Which the mean of blurring the error?

Comment (Gaiejj) [reply omnisafe/common/simmer_agent.py:106]: This blur operation is following the implementation of simmer RL in https://github.com/huawei-noah/HEBO/tree/master/SIMMER. I think blur the error can make the trainning process stable.

**Review by Gaiejj — COMMENTED** [review]
Please note that the performance of those algorithms can be seen at: https://wandb.ai/pku_rl/omnisafe/reports/OmniSafe-Mujoco-Velocity-Benchmarks--Vmlldzo0MDgzMDc3. I will open another PR to update the benchmark figure for all second order ``on-policy`` algorithms, as well as ``Simmer`` and ``PID``.

File: omnisafe/common/simmer_agent.py (hunk 4 above)
Comment (zmsn-2077) [reply omnisafe/common/simmer_agent.py:106]: get it

**Review by zmsn-2077 — APPROVED** [review]

**Pull request merged**